worker/sample.env

```diff
@@ -8,3 +8,4 @@ TOOL_CONTAINER_LABELS=""
 
 WORKFLOW_DATA_DIR="${PWD}/workflow_data/execution"
 TOOL_DATA_DIR="/data"
+GOOGLE_SERVICE_ACCOUNT_PATH="${ROOT_PATH}/artifact-key.json"
```

Comment:
@harini-venkataraman this should be actual service account json string instead of mounting it from file.

Reply:
@jaseemjaskp infact my suggestion is keep it mountable. We are already providing a SA key file for our on-prem customers. It would be easy in their case to mount thale same

@harini-venkataraman we will need to make a change in cloud platform repo to make it mountable. Do that as well

Reply:
@ritwik-g if we choose to keep it mountable, we should do the same everywhere else as well then. I believe we read this service account JSON env in `unstract/backend`, `llm-whisperer` and probably other places too

Reply:
@chandrasekharan-zipstack Adding, if we change the SA key file to customer, we can just change the file from the location and the envs can be untouched. I would also suggest mounting the key file.

Reply:
> > @jaseemjaskp we can keep it mountable also. But required change should be made in unstract cloud platform repo.
> > @harini-venkataraman also make this value empty. For OSS this value won't be present.
> 
> @ritwik-g Yes, for this reason have added a [condition](https://github.com/Zipstack/unstract/blob/2f474bccd5366b17b0fd8b169a49fb59df74b641/worker/src/unstract/worker/worker.py#L32) before the login. It doesnt login if the envs are not present.

@harini-venkataraman I think @ritwik-g meant to remove this from `sample.env` and instead add it in the cloud-platform repo. OSS users needn't have a default for this I believe

Reply:
@ritwik-g @chandrasekharan-zipstack Raised a PR to unstract-cloud-platform and removed from OSS
https://github.com/Zipstack/unstract-cloud-platform/pull/117